VLApi/Controllers/BooksController.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using VL.Shared.Data;
+using VL.Shared.Model;
+
+namespace VLApi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class BooksController : ControllerBase
+    {
+        private readonly ApplicationDbContext _context;
+
+        public BooksController(ApplicationDbContext context)
```

Comment:
Create a service, BookService that implements IBookService

Reply:
Can we review the reason for this?

Reply:
Service pattern, creating reusable code, containerizing your logic into a single service domain, dependency injection, list goes on...

Imagine you have another domain of your app where you have a dashboard and you need to display the count of books for each library at a higher level view, you wouldn't rewrite those methods in the DashboardService or whatever you would just use dependency injection to include that BookService and invoke one of the already created methods for example